caproto/sync/client.py

```diff
@@ -295,9 +320,9 @@ def subscribe(pv_name, priority=0, data_type=None, data_count=None,
     Examples
     --------
 
-    Define a subscription on the ``cat`` PV.
+    Define a subscription on the ``random_walk:x`` PV.
```

Comment:
This isn't really an important change; I just figured it might be helpful to use an example PV that we actually have an example IOC for instead of some arbitrary word.